controllers/user.go

```diff
@@ -35,6 +36,26 @@ func User(ctx context.Context, w http.ResponseWriter, r *http.Request) (status i
 	}
 }
 
+func GetUser(ctx context.Context, w http.ResponseWriter, r *http.Request) (status int, err error) {
```

Comment:
This function basically allows every user to query details of every other user, which must not be tolerated. There needs to be some sort of filter. What's the use case?